packages/contracts/src/contract-data.ts

```diff
@@ -0,0 +1,180 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
I tried switching the require statements to `import` but I couldn't get tslint to stop complaining. It said to add a list of `includes` to the config, which I tried, but still didn't work.